### PR TITLE
feat: 🎸 Added new props to Card component

### PR DESCRIPTION
**Pull request by mczapkowicz**
https://metakeen.atlassian.net/browse/FEE-262
FEE-667, FEE-668

### CODE REVIEW

File: packages/ui-core/src/components/card/card.component.tsx [hunk 1]
```diff
@@ -3,28 +3,30 @@ import { colors } from '@keen.io/colors';
...
-  border: ${(props) => props.border};
-  border-radius: ${(props) => props.borderRadius};
+  border: ${(props) => `${props.borderWidth}px solid ${props.borderColor}`};
+  border-radius: ${(props) => props.borderRadius}px;
   box-sizing: border-box;
   box-shadow: ${(props) =>
     props.hasShadow ? '0px 2px 4px 0px rgba(29,39,41,0.15)' : 'none'};
 
   ${(props) =>
     props.hasPadding &&
```
Comment (maciejrybaniec) [inline packages/ui-core/src/components/card/card.component.tsx:27]: Maybe we could get rid of `hasPadding` property in favor of just passing padding value. 
Then just set padding to `0` in this two places ? What do You think ?
Zrzut ekranu 2021-05-24 o 08 50 48

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

File: packages/ui-core/src/components/card/card.component.tsx [hunk 2]
```diff
@@ -35,11 +37,13 @@ export const Card = styled.div<Props>`
 `;
 
 Card.defaultProps = {
-  border: 'none',
-  borderRadius: '0px',
   backgroundColor: colors.white['500'],
   hasPadding: true,
   hasShadow: true,
+  padding: 20,
+  borderWidth: 20,
```
Comment (apryka) [inline packages/ui-core/src/components/card/card.component.tsx:44]: shouldn't it be `0` instead of `20`? by default we don't have any border in card

**Review by apryka — DISMISSED** [review]

File: packages/charts/src/components/legend/types.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export type LegendCardSettings = {
+  backgroundColor?: string;
+  border?: string;
```
Comment (apryka) [inline packages/charts/src/components/legend/types.ts:3]: is it still valid? 

` borderWidth?: number;  borderColor?: string;`

File: packages/charts/src/components/legend/types.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  borderRadius?: string;
+  hasShadow?: boolean;
+  hideOverflow?: boolean;
+  hasPadding?: boolean;
```
Comment (apryka) [inline packages/charts/src/components/legend/types.ts:7]: should be  `padding?: number;`

File: packages/charts/src/components/legend/types.ts (hunk 3 above)
Comment (mczapkowicz) [reply packages/charts/src/components/legend/types.ts:3]: Yes, it is valid. Thats why we decided to separate legend card settings type from regular card settings type, to not break compatibility with legend styling.

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**